plugins/txt/plugin/commands/txt_send_cmd.rb

```diff
@@ -0,0 +1,165 @@
+module AresMUSH
+  module Txt
+      class TxtSendCmd
+          include CommandHandler
+# Possible commands... txt name=message; txt =message; txt name[/optional scene #]=<message>
+
+          attr_accessor :names, :message, :scene_id, :scene, :txt, :txt_recipient
+
+        def parse_args
+          if (!cmd.args)
+            self.names = []
+
+          elsif (cmd.args.start_with?("="))
+            self.names = enactor.txt_last
```

Comment:
As a general rule, you don't want to use 'enactor' inside of `parse_args`, because it will throw an error if someone tries to use the command from the login screen.   `parse_args` is only good for parsing the actual arguments.

The only exceptions are `enactor_name` and `enactor_room`, which are special helpers that will return 'nil' if you're not logged in.

Probably not worth worrying about here; this handling is complex, so breaking it out would be annoying.  And the likelihood of anybody using the txt command when not logged in is small.  But just FYI for commands in general.

Reply:
Good to know. So you'd usually just define it down in handle? Or would you do Character.named(enactor_name) up here?

Reply:
In handle, yeah.